app/configurator/components/ui-helpers.ts

```diff
@@ -939,6 +909,44 @@ const divergingPaletteKeys = [
   "PuOr",
 ] as DivergingPaletteType[];
 
+const sequentialPaletteKeys = [
+  "blues",
+  "greens",
+  "greys",
+  "oranges",
+  "purples",
+  "reds",
+] as SequentialPaletteType[];
+
+const interpolatorByName = {
+  BrBG: interpolateBrBG,
+  PRGn: interpolatePRGn,
+  PiYG: interpolatePiYG,
+  PuOr: interpolatePuOr,
+  blues: interpolateBlues,
+  greens: interpolateGreens,
+  greys: interpolateGreys,
+  oranges: interpolateOranges,
+  purples: interpolatePurples,
+  reds: interpolateReds,
+};
+const defaultInterpolator = interpolatorByName["oranges"];
+
+export const getColorInterpolator = (
+  palette?: SequentialPaletteType | DivergingPaletteType
+): ((t: number) => string) => {
+  const interpolator = interpolatorByName[palette!] ?? defaultInterpolator;
+  // If the palette is sequential, we artificially clamp the value not to display too
+  // white a value
+  const isSequential = palette
+    ? // @ts-ignore
+      sequentialPaletteKeys.includes(palette)
+    : false;
```

Comment:
Nit: I wonder if here it wouldn't be enough to just do sequentialPaletteKeys.includes(palette), as even if it's undefined it should work the same way, no?

Reply:
Ah yes I added it because of typescript but then since there was the other error that types were incompatible between sequentialPaletteKeys ("blue" | "red" etc...) and palette (string), I added the typescript ignore.